### PR TITLE
Modified section on INSPIRE

### PR DESCRIPTION
**Pull request by alexanderkotsev**
The content is now migrated to the overviews folder.

### CODE REVIEW

**Comment by cvvergara** [comment]
@alexanderkotsev 
I finished moving files and doing small changes on the overview file.
But, when reading it carefully, it looks like there are two different things on the same file.
So here are some more tasks for when you have time:

### task 1
Can you break the file in 2:
* inspire_overview.rst
* ogc_overview.rst
  * follow the `@` convention of the logos and the names
  * don't forget the lines with the author name etc

### task 2:
Modify `projects_info.csv` and add a line similar to the one I added for inspire

### task 3:
I Used "Legislation" but if you think of a better name, please tell me so that I do some code changes

### Personal favor
Also, as a personal favor, can you break the paragraphs into lines of aprox 80 characters. Its difficult for me (that is why is personal) to move on such long lines using editor 'vi'

**Comment by alexanderkotsev** [comment]
@cvvergara I pushed my edits to the alexanderkotsev-master branch. 

I rephrased "Legislation" to "Geospatial Standards" as it is broader and better aligned with the actual content.

**Comment by cvvergara** [comment]
@alexanderkotsev 
One last thing I forgot to mention:
As its your first contribution:
- add a comment [like this one](https://github.com/OSGeo/OSGeoLive-doc/pull/258#issuecomment-313278899)
- add yourself to contributors.csv

File: doc/overview/inspire_overview.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+environmental policies. This European SDI will enable the sharing of
+environmental spatial information among public sector organisations, facilitate
+public access to spatial information and assist in policy-making across
+boundaries.  INSPIRE is based on the infrastructures for spatial information
+established and operated by the 28 Member States of the European Union. The
+Directive addresses 34 spatial data themes needed for environmental
+applications.  It came into force on 15 May 2007 and will be implemented in
+various stages, with full implementation required by 2021. INSPIRE is on a
+technical level based on international standards, mainly those developed by the
+OGC and ISO.
```
Comment (cvvergara) [inline doc/overview/inspire_overview.rst]: Maybe instead of only mentioning OGC you can put :doc:`OGC <ogc_overview>`

**Comment by alexanderkotsev** [comment]
@cvvergara I am already included in the contributors.csv and did confirm the license guidelines during the FOSSGIS event in Bonn.

**Pull request merged**